src/server/routes/generators/typescript.ts

```diff
@@ -55,6 +74,11 @@ export default async (fastify: FastifyInstance) => {
       reply.code(500)
       return { error: materializedViewsError.message }
     }
+    if (columnsError) {
+      request.log.error({ error: columns, request: extractRequestForLogging(request) })
```

Comment:
error: columns => error: columnsError?

```suggestion
      request.log.error({ error: columnsError, request: extractRequestForLogging(request) })
```

Reply:
Good catch!